src/dom-to-image-more.js

```diff
@@ -263,6 +263,10 @@
             function cloneStyle() {
                 copyStyle(window.getComputedStyle(original), clone.style);
 
+                if ((util.isChrome() || util.isSafari() ) && clone.style.marker && ( clone.tagName === 'line' || clone.tagName === 'path')) {
```

Comment:
I really don't like doing coarse "what browser is this" checks... is it possible to just always do these adjustments (e.g. don't care about if it's Chrome or Safari)

Reply:
@IDisposable  - sure, i just wanted to sync up changes in a central version of `dom-to-image` it is unclear which one will or should be the official version.